unstructured_inference/models/chipper.py

```diff
@@ -1,173 +1,502 @@
+import copy
 import os
-from typing import List, Optional
+from typing import Dict, List, Optional, Sequence, Tuple, Union
 
+import cv2
 import numpy as np
 import torch
+from huggingface_hub import hf_hub_download
 from PIL.Image import Image
-from transformers import (
-    AutoTokenizer,
-    DonutImageProcessor,
-    DonutProcessor,
-    VisionEncoderDecoderModel,
-)
-
-from unstructured_inference.inference.layoutelement import LocationlessLayoutElement
+from transformers import DonutProcessor, VisionEncoderDecoderModel
+from transformers.generation.logits_process import LogitsProcessor
+
+from unstructured_inference.constants import Source
+from unstructured_inference.inference.elements import Rectangle
+from unstructured_inference.inference.layoutelement import LayoutElement
 from unstructured_inference.models.unstructuredmodel import UnstructuredElementExtractionModel
+from unstructured_inference.utils import LazyDict
 
-MODEL_TYPES = {
+MODEL_TYPES: Dict[Optional[str], Union[LazyDict, dict]] = {
```

Comment:
should these be in constants?

Reply:
This is consistent with how the initialization parameters are handled for other models (detectron2, yolox, etc.). It probably could be done better but maybe it could be a separate PR?